components/match2/wikis/chess/chess_eco.lua

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
is it used outside of match2?
if yes imho this doesn't belong in the match2 folder

Reply:
only in Match2, not used on Infobox or other tables atm